src/config/dynamic-theme-fixes.config

```diff
@@ -1563,6 +1563,12 @@ code:first-of-type {
 #jira-issue-header, #jira-issue-header-actions {
     background-color: rgb(29, 32, 33) !important;
 }
+.ak-editor-content-area [colorname="Light green"] {
+  background-color: #073f29 !important;
+}
+.ak-editor-content-area [colorname="Light teal"] {
+  background-color: #003a42 !important;
+}
```

Comment:
Would it be able to use the form of `${...}` where `...` is replaced by the original hex color (see line 1573 for example)? This way dark reader can modify it on-the-fly and according to the user's settings.

Reply:
I'll try it next time I'm using it